Documentation/network/standalone.rst

```diff
@@ -0,0 +1,1610 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _standalone-gateway:
+
+*************************
+Cilium Standalone Gateway
+*************************
+
+Cilium has a standalone mode where it can be operated outside of Kubernetes
+as a standalone gateway. The gateway is designed to reside at the edge of the
+cluster, potentially replacing dedicated hardware components, and handling
+high-traffic load from a north-south direction going into the cluster(s).
+
+The Cilium Standalone Gateway includes functionality such as a high performance
+eBPF-based layer 4 load-balancer and a NAT46/64 gateway. There is a wide range
+of use cases, including:
+
+- Frontend DNS resolver
+- Load-balancer for legacy VM-based workloads,
+- As a gateway in front of IPv6 single stack Kubernetes clusters to allow for
+  interconnection with IPv4,
+- As a load-balancer for kubeapi-server,
+- As a replacement for legacy IPVS-based load-balancers,
+- And many more!
+
+Cilium provides a programmable API for its agent which is exposed through a
+unix domain socket on the local node such that it can be integrated into
+third party management software on the node.
+
+This guide explains how to provision Cilium as a standalone gateway along with
+various use-cases and configuration examples.
+
+Quick-Start
+===========
+
+Given the Cilium Standalone Gateway can be operated outside of a Kubernetes
+cluster, this guide provides minimal Docker-based step-by-step instructions.
+
+As a one-time prerequisite, ensure that the BPF file system is mounted under
+``/sys/fs/bpf``. If this is not the case already, a mount unit file can be set
+up under ``/etc/systemd/system/sys-fs-bpf.mount`` ...
+
+.. parsed-literal::
```

Comment:
`parsed-literal` enables additional preprocessing such as substituting links in the output based on predefined macros like `\ |SCM_WEB|`. This should ideally be formatted with some systemd syntax highlighting, but I'd guess that doesn't exist in sphinx.

Easiest path is just to use plain formatting of a text block:

```suggestion
::
```